examples/src/Components/AgentMetadata.tsx

```diff
@@ -1,18 +1,18 @@
 import React from "react";
-import { AgentData } from "../../type-declarations/simularium/types";
+import { AgentData } from "../../../type-declarations/simularium/types";
 
 interface AgentMetadataProps {
     agentData: AgentData;
 }
 
 const AgentMetadata = ({ agentData }: AgentMetadataProps): JSX.Element => {
-
     const getContents = () => {
         if (agentData.instanceId === -1) {
-            return <div>No agent selected</div>;
+            return <div className="ui-container">No agent selected</div>;
```

Comment:
Existing custom components get this wrapper to show what they contain, goal is to factor all the loose UI html into components and then get it into a scheme.